scripts/diag/tmux-server-contention.py

```
@@ -0,0 +1,261 @@
+        armed_socks.append(solo_sock)
+        solo_pane, solo_cap = arm_up(solo_sock, "solo")
+
+        for i in range(1, a.samples + 1):
+            loads.append(os.getloadavg()[0])
+            for sends, sock, pane, pfx in (
+                (busy_sends, a.busy_socket, busy_pane, "B"),
+                (solo_sends, solo_sock, solo_pane, "S"),
+            ):
+                tok = f"CT{pfx}{i:05d}"
+                t0 = time.time()
+                subprocess.run([TMUX_BIN, "-S", sock, "send-keys", "-t", pane, tok, "Enter"],
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)